201-vmss-master-slave-customscript/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
```

Comment:
Must be:   "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",